### ISSUE TITLE
[bug report] Jasmine Typing errors

### ISSUE THREAD

**Issue by andyrue** [valor-software/ng2-dragula#896]
**REQUIRED: Before filing a bug report**

Change each `[ ]` to `[x]` when you have done it.

- [x] My issue title starts with `[bug report]`
- [x] I have read the [README](https://github.com/valor-software/ng2-dragula), especially the ['classic blunders' section](https://github.com/valor-software/ng2-dragula#classic-blunders), and the [dragula docs](https://github.com/bevacqua/dragula).
- [x] I have searched through the [`ng2-dragula` issues](https://github.com/valor-software/ng2-dragula/issues) for related problems, including closed issues.
- [x] I have browsed through the [issues labeled "future reference"](https://github.com/valor-software/ng2-dragula/issues?utf8=%E2%9C%93&q=is%3Aissue+label%3A%22future+reference%22) for problems that have been solved before or have a known workaround.
- [x] I am using the latest version of `ng2-dragula`.

**Describe the bug (required)**

I'm in the process of updating to Angular 6 and with that ng2-dragula v2.0.2. When I run `ng serve` I get:

    ERROR in node_modules/ng2-dragula/dist/jasmine.d.ts(2,38): error TS1147: Import declarations in a namespace cannot reference a module.
    node_modules/ng2-dragula/dist/jasmine.d.ts(2,38): error TS2307: Cannot find module 'testdouble'.
    node_modules/ng2-dragula/dist/testdouble-jasmine.d.ts(3,33): error TS2709: Cannot use namespace 'CustomMatcherFactories' as a type.

**To Reproduce (required)**

This may be isolated to me, but the way I reproduce it is to install the latest version of Angular and ng2-dragula@2.0.2 in my project. Then run `ng serve`

**Versions (required)**

Please state which versions of the following packages you have installed:

 - `@angular/core`: 6.1.4
 - `ng2-dragula`: 2.0.2

**Comment by cormacrelf** [comment]
Typescript version please

**Comment by andyrue** [comment]
typescript: 2.9.2

**Comment by andyrue** [comment]
I apologize, this turned out to be my fault. After installing the latest update 2.0.4 and still having the problem, I continued digging and realized I was including `"../node_modules/ng2-dragula/**/*.ts",` in my tsconfig.app.json file. I had only checked my tsconfig.json file for stray entries. I was including the folder before because I was using a forked version and installing from git. Sorry for the trouble.
